pkg/hs/urbit-king/lib/Urbit/King/Main.hs

```diff
@@ -693,6 +703,11 @@ main = do
     for_ [Sys.sigTERM, Sys.sigINT] $ \sig -> do
       Sys.installHandler sig (Sys.Catch onKillSig) Nothing
 
+  setRLimits = do
+    nofiles <- Sys.getResourceLimit Sys.ResourceOpenFiles
+    Sys.setResourceLimit Sys.ResourceOpenFiles
+      nofiles { Sys.softLimit = Sys.ResourceLimit 10240 }
```

Comment:
We still want the max number of files on linux, which should be accurately reported by `getResourceLimit` and can be higher than this. It just needs to be hardcoded on MacOS.